plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -52,6 +55,50 @@ protected function get_settings_for_default_section() {
 			'https://wordpress.org/plugins/wp-mail-logging/',
 			'https://woocommerce.com/document/email-faq'
 		);
+
+		/* translators: %s: Nonced email preview link */
+		$email_template_description = sprintf( __( 'This section lets you customize the WooCommerce emails. <a href="%s" target="_blank">Click here to preview your email template</a>.', 'woocommerce' ), wp_nonce_url( admin_url( '?preview_woocommerce_mail=true' ), 'preview-mail' ) );
+		$logo_image                 = array(
+			'title'       => __( 'Header image', 'woocommerce' ),
+			'desc'        => __( 'Paste the URL of an image you want to show in the email header. Upload images using the media uploader (Media > Add New).', 'woocommerce' ),
+			'id'          => 'woocommerce_email_header_image',
+			'type'        => 'text',
+			'css'         => 'min-width:400px;',
+			'placeholder' => __( 'N/A', 'woocommerce' ),
+			'default'     => '',
+			'autoload'    => false,
+			'desc_tip'    => true,
+		);
+		$header_alignment           = null;
+
+		if ( FeaturesUtil::feature_is_enabled( 'email_improvements' ) ) {
+			$email_template_description = sprintf( __( 'Customize your WooCommerce email template and preview it below.', 'woocommerce' ) );
+			$logo_image                 = array(
+				'title'       => __( 'Logo', 'woocommerce' ),
+				'desc'        => '',
+				'id'          => 'woocommerce_email_header_image',
+				'type'        => 'email_image_url',
+				'css'         => 'min-width:400px;',
+				'placeholder' => __( 'N/A', 'woocommerce' ),
+				'default'     => '',
+				'autoload'    => false,
+				'desc_tip'    => true,
+			);
+			$header_alignment           = array(
+				'title'    => __( 'Header alignment', 'woocommerce' ),
+				'id'       => 'woocommerce_email_header_alignment',
+				'desc_tip' => '',
+				'default'  => 'base',
```

Comment:
Should this be `left`?

Reply:
Right. I'll fix it. What do you think of this point from the PR description, by the way:
> Note that by default logo and header alignment are different (center vs left), so it's a minor breaking change.

Reply:
I think it's okay. While it is a breaking change, it's only breaking for those who enable the feature flag. Once we launch it for everyone, we'll also have new email styles.